contrib/deploy-onefuzz-via-azure-devops/deploy-onefuzz.yml

```diff
@@ -74,7 +82,12 @@ stages:
             script: |
               set -ex
               az login --service-principal -u $(ONEFUZZ_SERVICE_URL) -p $(AZURE_CLIENT_SECRET) --tenant $(AZURE_TENANT_ID)
-              python deploy.py --client_id $(AZURE_CLIENT_ID) --client_secret $(AZURE_CLIENT_SECRET) $REGION $RESOURCE_GROUP_NAME $ONEFUZZ_INSTANCE_NAME $CONTACT_EMAIL_ADDRESS
+              if [ -z ${DEPLOY_ARGS+x} ]
+              then
+                python deploy.py --client_id $(AZURE_CLIENT_ID) --client_secret $(AZURE_CLIENT_SECRET) $REGION $RESOURCE_GROUP_NAME $ONEFUZZ_INSTANCE_NAME $CONTACT_EMAIL_ADDRESS
+              else
+                python deploy.py --client_id $(AZURE_CLIENT_ID) --client_secret $(AZURE_CLIENT_SECRET) $REGION $RESOURCE_GROUP_NAME $ONEFUZZ_INSTANCE_NAME $CONTACT_EMAIL_ADDRESS $(DEPLOY_ARGS)
```

Comment:
Correct me if I am wrong. I think we don't need to put in `if` statement. We can just use the second one. If `DEPLOY_ARGS` id empty it will be same the above `deploy` statement.